source/jsonnet/england-wales/lib/transforms.libsonnet

```diff
@@ -0,0 +1,104 @@
+local formatFirstPersonName = {
```

Comment:
As this is often used where a second person doesn't exist, I think it should be named `formatPersonName`.